src/screens/Exchange/Sell.js

```diff
@@ -1,24 +1,100 @@
 // @flow
 
-import React from "react";
-import { View, StyleSheet } from "react-native";
+import React, { useCallback, useState, useEffect } from "react";
+import { View, StyleSheet, Platform } from "react-native";
 import SafeAreaView from "react-native-safe-area-view";
 import { useTranslation } from "react-i18next";
 import { useNavigation, useTheme } from "@react-navigation/native";
-import { NavigatorName } from "../../const";
+import type {
+  Account,
+  AccountLike,
+  CryptoCurrency,
+  TokenCurrency,
+} from "@ledgerhq/live-common/lib/types";
+import { useRampCatalog } from "@ledgerhq/live-common/lib/platform/providers/RampCatalogProvider";
+import { currenciesByMarketcap } from "@ledgerhq/live-common/lib/currencies";
+import { getAccountCurrency } from "@ledgerhq/live-common/lib/account/helpers";
+import { isAccountEmpty } from "@ledgerhq/live-common/lib/account";
 import extraStatusBarPadding from "../../logic/extraStatusBarPadding";
 import TrackScreen from "../../analytics/TrackScreen";
-import LText from "../../components/LText";
-import ExchangeIcon from "../../icons/Exchange";
 import Button from "../../components/Button";
-import PoweredByCoinify from "./PoweredByCoinify";
+import { NavigatorName, ScreenName } from "../../const";
+import { useRampCatalogCurrencies } from "./hooks";
+import SelectAccountCurrency from "./SelectAccountCurrency";
+import { track } from "../../analytics";
 
 const forceInset = { bottom: "always" };
 
-export default function Buy() {
-  const { colors } = useTheme();
+export default function OffRamp() {
   const { t } = useTranslation();
+  const { colors } = useTheme();
   const navigation = useNavigation();
+  const rampCatalog = useRampCatalog();
+  const allCurrencies = useRampCatalogCurrencies(rampCatalog.value.offRamp);
```

Comment:
If the provided it not loaded at this time, value will be undefined. Make sure the data is loaded by checking value is defined

Reply:
fix uploaded